gbpservice/neutron/nsf/config_agent/firewall.py

```diff
@@ -0,0 +1,154 @@
+import sys
```

Comment:
This file should be under nsf/config_agent/modules. This should be consistent with the dir structure for lifecycle_manager and configurator. Check with JAM [JagAhmMag]